src/samples/Uno.Material.Samples/Uno.Material.Samples.Droid/Resources/values/Styles.xml

```diff
@@ -10,5 +10,9 @@
 		
 		<!-- SplashScreen -->
 		<item name="android:windowBackground">@drawable/Splash</item>
+
+		<!-- Color style for toggle switch -->
```

Comment:
that's in the sample, is it required by the consuming app?

Reply:
added to readme